src/images.js

```diff
@@ -0,0 +1,21 @@
+const images = {
```

Comment:
I would add the paths for icons which are also in `public`, if we're having a js file store our resource paths.

Reply:
everything is in public now

Reply:
great! Tyvm!